### PR TITLE
Temporarily disable checking IPv6 BGP routing on neighbors for warm-reboot

### PR DESCRIPTION
**Pull request by saiarcot895**
### Description of PR

Due to sonic-net/sonic-buildimage#12853, IPv6 BGP is partially broken, where some IPv6 BGP prefixes aren't advertised up to T1 neighbors. In an effort to get the warm reboot PR checker re-enabled, disable checking the IPv6 BGP routing for now.

Once that PR is either fixed or reverted, revert this PR.

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

### CODE REVIEW

**Review by kellyyeh — DISMISSED** [review]

File: ansible/roles/test/files/ptftests/arista.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def run(self):
             'interfaces']['Port-Channel1']['lastStatusChangeTimestamp']
         samples[cur_time] = sample
```
Comment (vaibhavhd) [inline ansible/roles/test/files/ptftests/arista.py:150]: May be add a TODO in comments to revert this change after FRR issue (w/ link) is fixed.

**Review by vaibhavhd — DISMISSED** [review]

File: ansible/roles/test/files/ptftests/arista.py (hunk 1 above)
Comment (saiarcot895) [reply ansible/roles/test/files/ptftests/arista.py:150]: Done.

**Review by vaibhavhd — APPROVED** [review]
